TechnicoBackend/Services/UserService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using TechnicoBackend.Repositories;
+using TechnicoBackend.Models;
+
+namespace TechnicoBackend.Services
+{
+    public class UserService
+    {
+        private readonly UserRepository _userRepository;
```

Comment:
Consider initializing an IUserRepository here instead of UserRepository, which would be mapped accordingly using proper Dependency Injection in Program.cs.